### ISSUE TITLE
FAQ: Does the chains.json support load balancers? 

### ISSUE THREAD

**Issue by jeremybeal11** [pokt-network/pocket-core#1108]
### PURPOSE
This issue is meant to serve as a reference to answer a common question for understanding whether you can add a load balancer address in your chains.json file
### GUIDELINES
- If FAQ does not comprehensively cover the issue, community comments and contributions are welcome
- Although the issue is closed, FAQ issues are meant to be maintained and edited over time
## Does the chains.json support load balancers? 
In short, yes. All you need to do is specify the address and the port in the `url` param in your chains.json. 

### NOTES
Add any additional notes here
